tests/Etc/Etc.sol

```diff
@@ -32,4 +32,11 @@ contract Contract {
     uint sum;
     for ( i = 0; i < 10; i++ ) sum   +=  i;
   }
+
+  function fun(uint256 a) returns (uint) {
+    if (something) foo();
+    // comment
```

Comment:
Shouldn't there be an updated snapshot for this too?

Reply:
nope as it throws an error now as per:

https://github.com/prettier-solidity/prettier-comments#how-to-use-it-w-prettier-plugin-solidity

basically the `comments.js` I've extracted from `prettier` knows how to scan their AST but not the one we are using (this is my assumption)